src/orm/entities/account.element.type.enum.ts

```diff
@@ -0,0 +1,21 @@
+export enum AccountElementTypeEnum {
+  PUBKEY = 'Pubkey',
+  BOOL = 'bool',
+  U64 = 'u64',
+  U128 = 'u128',
+  I64 = 'i64',
+  I128 = 'i128',
+  U8 = 'u8',
+  U16 = 'u16',
+  U32 = 'u32',
+  I8 = 'i8',
+  I16 = 'i16',
+  I32 = 'i32',
+  F32 = 'f32',
+  F64 = 'f64',
+  ENUM = 'Enum',
+  STRUCT = 'Struct',
+  TN = '[T; N]',
+  STRING = 'String',
+  VECT = 'Vec<T>',
+}
```

Comment:
also, enum should not be in the entities folder. maybe the better place for it is **src/app/common** or **src/common?**